openmdao_book/features/model_visualization/n2_basics/n2_basics.ipynb

```diff
@@ -0,0 +1,229 @@
+{
```

Comment:
<p>I'd recommend using a bash block for this, and then put the actual command with "!" in a hidden cell.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OpenMDAO/OpenMDAO_Book/pull/104/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>